dashboard/test/ui/features/hour_of_code/ml/ml_hoc.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Oceans ML HoC
 
-  @no_circle @no_mobile @no_safari
+  @no_circle @no_mobile @no_safari @no_ie
```

Comment:
not blocking as this should be a quick change, but couldn't all these settings be set at once at the top of this file?